drivers/spi/Kconfig.stm32

```diff
@@ -13,6 +13,13 @@ menuconfig SPI_STM32
 
 if SPI_STM32
 
+config SPI_STM32_FIFO
```

Comment:
As explained in other comments, enabling/disabling the SPI FIFO is done now per each instance, and from devicetree.